code/core/src/starter/DefaultMainController.java

```diff
@@ -30,6 +46,25 @@ protected void frame() {
         }
     }
 
+    private void checkMovingKeys() {
+        if (Gdx.input.isKeyPressed(Input.Keys.U)) {
+            Vector3 position = camera.position;
+            camera.setFocusPoint(new Point(position.x, position.y + 1));
+        }
+        if (Gdx.input.isKeyPressed(Input.Keys.J)) {
+            Vector3 position = camera.position;
+            camera.setFocusPoint(new Point(position.x, position.y - 1));
+        }
+        if (Gdx.input.isKeyPressed(Input.Keys.H)) {
+            Vector3 position = camera.position;
+            camera.setFocusPoint(new Point(position.x - 1, position.y));
+        }
+        if (Gdx.input.isKeyPressed(Input.Keys.K)) {
+            Vector3 position = camera.position;
+            camera.setFocusPoint(new Point(position.x + 1, position.y));
+        }
+    }
+
     @Override
     public void onLevelLoad() {}
```

Comment:
Ich würde den start-Fokus Point der Kamera auf den Level-Start legen. Dann sieht man beim ersten Start auch direkt was vom Level und wundert sich nicht, warum da nur schwarz ist. 

```suggestion
    public void onLevelLoad() {
        camera.setFocusPoint(levelAPI.getCurrentLevel().getStartTile().getCoordinate().toPoint());
    }
```